src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arr = sourceString
+    .split(';')
+    .map((el) => el.trim())
```

Comment:
[CODE STYLE] - Avoid variable namings like `arr`, as it decreases code readability. Choose a more descriptive name that reflects the content or purpose of the array.